README.md

```diff
@@ -100,7 +100,22 @@ To uninstall the chart:
 
 #### Deployment Probes Paramaters
 
+##### Startup Probe
+StartupProbe indicates that the Pod has successfully initialized. If specified, no other probes are executed until this completes successfully.
+
+| Name                     | Description                                                                                  | Value           |
+| ------------------------ | -------------------------------------------------------------------------------------------- | --------------- |
+| deployment.startupProbe.enabled | Enabled startup probe                                                                  | true       |
+| deployment.startupProbe.failureThreshold | When a probe fails, Kubernetes will try failureThreshold times before giving up.                                                                  | 3      |
```

Comment:
this value should be bit higher. thats where this probe delivers value that  it does not fail immediately during startup. I would say 10

Reply:
I had it bumped 30 (5 min) in the readme as it was that in the values file. Is 30 ok or do you like me to set it to 10 (100 sek)?

Reply:
30 should be fine